roles/analytics-dashboard/tasks/main.yml

```diff
@@ -83,9 +109,6 @@
     become_method: sudo
     become_user: root
 
-# Enable customization by uncommenting the following and adding the changes to the custom.yml
-      # - import_tasks: custom.yml
-
```

Comment:
Keep this instruction

Reply:
fixed https://github.com/wso2/ansible-ei/pull/30/commits/bf084ed7e012838529727ccbb5adb33110a47243